lib/driver.js

```diff
@@ -142,8 +145,7 @@ class XCUITestDriver extends BaseDriver {
   }
 
   get driverData () {
-    // TODO fill out resource info here
-    return {};
+    return this._driverData;
```

Comment:
I'm not sure what you mean by that.

But here it needs to access `this._driverData`, since it is inside the getter. But other than the initialization, this should be the _only_ place where `_driverData` is touched.